src/software/firmware/srcs/activation.cpp

```diff
@@ -0,0 +1,50 @@
+/******************************************************************************
+ * @author Makers For Life
+ * @copyright Copyright (c) 2020 Makers For Life
+ * @file activation.h
+ * @brief Breathing activation related functions
+ *****************************************************************************/
+
+// INCLUDES ===================================================================
+
+// Associated header
+#include "../includes/activation.h"
+
+// External libraries
+
+// Internal
+#include "../includes/buzzer.h"
+
+const uint32_t SECOND_STOP_MAX_DELAY_MS = 5000;
+
+// INITIALISATION =============================================================
+
+ActivationController activationController;
+
+// FUNCTIONS ==================================================================
+
+ActivationController::ActivationController() : m_state(STOPPED), m_timeOfLastStopPushed(0) {}
+
+void ActivationController::onStartButton() { m_state = RUNNING; }
```

Comment:
If previous state was `STOPPED`, maybe we should do a `Buzzer_Boot_Start()`?

Reply:
Actually there is no need.